src/cpu/x64/utils/jit_io_helper.cpp

```diff
@@ -802,9 +802,9 @@ template <typename Vmm>
 void jit_io_helper_t<Vmm>::saturate(const Vmm &vmm) {
     assert(saturation_conf_.has_value() && "Config for saturation is not set.");
 
-    host_->saturate_cvt_f32(vmm,
-            Vmm(saturation_conf_->vreg_zero_saturation_idx_),
-            Vmm(saturation_conf_->vreg_saturation_ubound_idx_), data_type_);
+    host_->sub(host_->rsp, 64);
+    host_->uni_vmovups(host_->ptr[host_->rsp], vmm);
+    host_->add(host_->rsp, 64);
```

Comment:
This is not how the stack works. The process must use the stack when it's accessible. By adding back it becomes inaccessible as it was before sub.